sphinx/user-guide/installation.rst

```diff
@@ -0,0 +1,427 @@
+recommendation and store Rose configuration metadata under the
+``rose-meta/`` directory of their source tree, you can:
+
+* Check out a working copy for each sub-directory under the
+  ``rose-meta/`` directory.
+* Set up a crontab job to regularly update the working copies.
+
+For example, suppose you want to deploy Rose configuration metadata
+under ``/etc/rose-meta/`` at your site. You can do::
+
+   # Deployment location
+   DEST='/etc/rose-meta'
+   cd "${DEST}"
+
+   # Assume only Rose metadata configuration directories under "rose-meta/"
+   URL1='https://somehost/foo/main/trunk/rose-meta'
+   URL2='https://anotherhost/bar/main/trunk/rose-meta'
+   # ...
+
+   # Checkout a working copy for each metadata configuration directory
+   for URL in "${URL1}" "${URL2}"; do
+     for NAME in $(svn ls "${URL}"); do
+         svn checkout -q "${URL}/${NAME}"
+     done
+   done
+
+   # Set up a crontab job to update the working copies, e.g. every 10 minutes
+   crontab -l || true >'crontab.tmp'
+   {
+     echo '# Update Rose configuration metadata every 10 minutes'
+     echo "*/10 * * * * svn update -q ${DEST}/*"
+   } >>'crontab.tmp'
+   crontab 'crontab.tmp'
+   rm 'crontab.tmp'
+
+   # Finally add the root level "meta-path" setting to site's "rose.conf"
+   # E.g. if Rose is installed under "/opt/rose/":
+   {
+     echo '[]'
+     echo "meta-path=${DEST}"
+   } >>'/opt/rose/etc/rose.conf'
+
+.. tip::
+   See also :ref:`app-meta-loc`.
+
+
+Configuring Rose Bush
+---------------------
+
+Rose Bush provides an intranet web service at your site for users to
+view their suite logs using a web browser. Depending on settings at
+your site, you may or may not be able to set up this service.
+
+You can start an ad-hoc Rose Bush web server by running::
+
+   setsid /path/to/rose/bin/rose bush start 0</dev/null 1>/dev/null 2>&1 &
+
+You will find the access and error logs under ``~/.metomi/rose-bush*``.
+
+Alternatively you can run the Rose Bush web service under Apache
+``mod_wsgi``. To do this you will need to set up an Apache module
+configuration file (typically in ``/etc/httpd/conf.d/rose-wsgi.conf``)
+containing the following (with the paths set appropriately)::
+
+   WSGIPythonPath /path/to/rose/lib/python
+   WSGIScriptAlias /rose-bush /path/to/rose/lib/python/rose/bush.py
+
+Use the Apache log at e.g. ``/var/log/httpd/`` to debug problems.
+See also Configuring Rosie Server below.
+
+.. TODO ---
+
+
+Configuring a Rosie Server
+--------------------------
+
+You should only need to configure and run your own Rosie service if you do
+not have access to Rosie services on the Internet, or if you need a
+private Rosie service for your site. Depending on settings at your
+site, you may or may not be able to set up this service.
+
+You will need to select a machine to host the Subversion repositories.
+This machine will also host the web server and databases.
+
+Login to your host, create one or more Subversion FSFS repositories.
```

Comment:
Maybe add an external link here to clarify the nature of 'Subversion FSFS' and/or to external tutorial/instructions for how to create those repositories? Seems a bit technical so some readers might appreciate that.